### PR TITLE
Fixes #36155 - Include example in tooltip for Target URL

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/WebhookFormTabs.js [hunk 1]
```diff
@@ -48,7 +48,11 @@ const WebhookFormTabs = ({
           type="text"
           required
           label={__('Target URL')}
-          labelHelp={__('Target URL that should be called by Foreman')}
+          labelHelp={
+            __('Target URL that should be called by Foreman (ERB allowed).') +
+            ' ' +
+            sprintf(__('Example: %s'), 'https://host.example.com/inventory/<%= @object.id %>')
```
Comment (adamruzicka) [inline webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/WebhookFormTabs.js:54]: til the url is an erb template

**Comment by adamruzicka** [comment]
It would maybe be more readable if the example was on its own line, but I don't insist on it

**Comment by ofedoren** [comment]
> It would maybe be more readable if the example was on its own line, but I don't insist on it

Totally agree on that, but I didn't found a way how to do it via `labelHelp`. I've tried to inject `\n` and `&#13;&#10;`, but without success :/

**Comment by adamruzicka** [comment]
Not sure if this is a good way to go, but labelHelp seems to accept a react fragment, so you can pass in a fragment with two divs and they'll render as separate lines.

**Comment by ofedoren** [comment]
Thanks, @adamruzicka, it did the trick :facepalm:

**Review by adamruzicka — APPROVED** [review]
Lgtm

**Pull request merged**